source/scale/elasticsearch.rst

```diff
@@ -26,13 +26,82 @@ Set up an Elasticsearch server
 
 The set up process for the Elasticsearch server is documented in the `official Elasticsearch documentation <https://www.elastic.co/guide/en/elasticsearch/reference/7.17/setup.html>`__.
 
-.. important::
-  You must install the `ICU Analyzer Plugin <https://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-icu.html>`__ when setting up Elasticsearch for Mattermost.
+1. Download and install the RPM package manager by opening a terminal window and running the following command:
+
+  .. code-block:: none
+
+    wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-7.10.2-x86_64.rpm
+    sudo rpm --install elasticsearch-7.10.2-x86_64.rpm
+
+2. Set up Elasticsearch with systemd by running the following commands:
+
+  .. code-block:: none
+
+    sudo /bin/systemctl daemon-reload
+    sudo /bin/systemctl enable elasticsearch.service
+    sudo systemctl start elasticsearch.service
+
+3. Confirm Elasticsearch is working on the server by running the following command:
+
+  .. code-block:: none
+
+    curl localhost:9200
+
+4. Get your network interface name by running the following command:
+
+  .. code-block:: none
+
+    ip addr
+
+5. Edit the Elasticsearch configuration file in vi by running the following command:
+
+  .. code-block:: none
+
+    vi /etc/elasticsearch/elasticsearch.yml
+
+6. In this file, replace the ``network.host`` value of ``_eth0_`` with your network interface name, and save your changes.
+
+7. Restart Elasticsearch by running the following commands:
+
+  .. code-block:: none
+
+    sudo systemctl stop elasticsearch
+    sudo systemctl start elasticsearch
+
+8. Confirm the ports are listenings by running the following command:
+
+  .. code-block:: none
+
+    netstat -plnt
+
+  You should see the following ports, including  the ones listening on ports 9200 and 9300. Confirm these are listening on your server's IP address. 
+
+9. Create an Elasticsearch directory within the ``/data`` mount and give it the proper permissions by running the following commands:
+
+  .. code-block:: none
+
+    cd /data
```

Comment:
If we stick with these docs, we'll need to adjust this. This was specifically for a customer who uses a specific mount to `/data`. Not a hard adjustment

Reply:
Thanks, @coltoneshaw! I've updated the draft copy accordingly.